functions/package.json

```diff
@@ -13,19 +13,25 @@
   "dependencies": {
     "body-parser": "^1.19.0",
     "express": "^4.14.1",
-    "firebase-admin": "~7.0.0",
+    "firebase-admin": "^6.5.1",
```

Comment:
Because the firebase-admin dependency in firebase-functions-helper is still in 6.5.1, downgrade to package to the same version solved the namespace problem.

Reply:
Thanks @mblonyox